UnitTests/Features/Event/CancelParticipation/CancelParticipationHandlerTests.cs

```diff
@@ -0,0 +1,58 @@
+using UnitTests.Common.Repositories;
+using UnitTests.Common.UnitOfWorkImp;
+using UnitTests.Features.Event;
+using UnitTests.Features.Guest;
+using ViaEventAssociation.Core.Application.AppEntry.Commands.Guest;
+using ViaEventAssociation.Core.Application.Handlers.Features.Guest;
+using ViaEventAssociation.Core.Domain;
+using ViaEventAssociation.Core.Domain.Common;
+using Xunit;
+
+namespace UnitTests.Features.CancelParticipation;
+
+public class CancelParticipationHandlerTests
+{
+    private readonly IEventRepository eventRepo = new EventRepositoryFake();
+    private readonly IGuestRepository guestRepo = new GuestRepositoryFake();
+    private readonly IUnitOfWork uow = new UnitOfWork();
+    // Success
+    [Fact]
+    public void CancelParticipation_EventInReadyStatus_ReturnsSuccess()
```

Comment:
Trolse said that we need test dor commands and test for handlers try to separate them.